pandas/core/indexes/datetimes.py

```diff
@@ -2137,6 +2165,9 @@ def cdate_range(start=None, end=None, periods=None, freq='C', tz=None,
     -------
     rng : DatetimeIndex
     """
+    warnings.warn("cdate_range is deprecated and will be removed in a future "
+                  "version, instead use bdate_range(..., freq='{freq}')"
```

Comment:
bdate_range -> pd.bdate_range (to make it clear that bdate_range is top-level, as opposed to cdate_range)